week-2/6-playing-computer.js

```diff
@@ -6,13 +6,20 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  1. This program throws an error. Why? (If you can't find it, try executing it). 
+   Solution:  Console log b
+  2. Remove the line that throws the error. 
+   Solution:  Line 33 removed
+  3. What is printed to the console? 
+  Solution: values of x, a,  d and e
```

Comment:
I think this question isn't quite clear: we wanted to know what the actual values are when you run the code 🙂So for example, 5, 4, 3 (not the correct answer obviously!)

Could you please comment here/change your answer to show that?